api/src/main/java/org/pmiops/workbench/dataset/DataSetServiceImpl.java

```diff
@@ -93,6 +93,15 @@
           Domain.PROCEDURE,
           Domain.PHYSICAL_MEASUREMENT_CSS);
 
+  private static final ImmutableList<Domain> DOMAIN_NO_CONCEPT_SET =
```

Comment:
nit: would name this something like `DOMAINS_WITHOUT_CONCEPT_SETS`

Reply:
Done